ui/lib/kv/addon/components/page/secret/details.hbs

```diff
@@ -6,7 +6,7 @@
 <KvPageHeader @breadcrumbs={{@breadcrumbs}} @pageTitle={{@path}}>
   <:syncDetails>
     {{#if this.syncStatus}}
-      <Hds::Alert data-test-sync-alert @type="page" @color="neutral" as |A|>
+      <Hds::Alert data-test-sync-alert @type="inline" class="has-top-margin-s has-bottom-margin-m" @color="neutral" as |A|>
```

Comment:
Consulted design and this should have been an `inline` alert all along
<img width="1064" alt="Screenshot 2024-01-09 at 8 22 57 PM" src="https://github.com/hashicorp/vault/assets/68122737/9cd8937d-97d4-436a-b540-846e4ef94b75">